### DISCUSSION TITLE
Webcam device - RTSP authentication

### DISCUSSION THREAD

**Discussion by niho06** [bluenviron/mediamtx#2280] [General]
It is possible to add a username and password to the RTSP link that it offers when a USB type webcam is integrated.

I see that it maintains video encryption but not authentication.

grateful if someone can tell me the configuration or if it is not possible

greetings.

**Comment by aler9** [comment]
Hello, you can setup authentication by following instructions here:

https://github.com/bluenviron/mediamtx#authentication

**Reply by niho06** [reply]
Thank, but just error.
![image](https://github.com/bluenviron/mediamtx/assets/122945130/1e0d30c5-35f9-48ed-a948-568cd38f32f8)

I think I'm doing something wrong

![image](https://github.com/bluenviron/mediamtx/assets/122945130/fc2eab97-8082-4dd5-b753-df40b23af035)

Can you tell me what should I add to make it work?

[a comment by an automated account is not shown]
